src/js/components/Form/stories/ValidateDisabled.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+
+import {
+  Grommet,
+  Box,
+  Button,
+  Form,
+  FormField,
+  Text,
+  TextInput,
+  RadioButtonGroup,
+} from 'grommet';
+
+export const ValidateDisabled = () => {
+  const [formValues, setFormValues] = useState({ switch: 'Enable' });
+  const [validationResults, setValidationResults] = useState({});
+  const updateFormFields = (newValue) => {
+    // If we're disabling the field, clear what's in it
+    if (newValue === 'Disable') {
+      setFormValues({ firstName: 'a', switch: newValue });
+    }
```

Comment:
I think this part could be removed from the story so that it doesn't force the disabled value to be 'a' and various values can be tested with disabled

Reply:
I added this otherwise that state is not being set for the form value. then we run into the issue before where it was not getting updated

Reply:
we decided to keep this as is!

Reply:
I would say this story is not intended to be merged, rather a convenient means for reviewers to test. Is that your perspective as well? Or do you see this adding value?